pkg/util/iptables/iptables.go

```diff
@@ -783,3 +789,90 @@ func isResourceError(err error) bool {
 	}
 	return false
 }
+
+// ParseError records the payload when iptables restore fails
```

Comment:
nitpick: something like "ParseError records the payload when iptables reports an error parsing its input."

Reply:
sorry, my expression is not quite accurate 😅